File: Assignment/Module-1(fundamentals/p5.cpp
#include<stdio.h>
main()
{
	float base,height,area;
	printf("Enter the base length of the triangle:");
	scanf("%f",&base);
	printf("Enter the height of the triangle: ");
	scanf("%f",&height);
	area=(base*height)/2;
	printf("the area of the triangle is: %.2f square units\n",area);
}

File: Assignment/flutter dart assignment module-1/p5.dart
import 'dart:io';

void main() {

  stdout.write("Enter the length of the base of the triangle: ");
  double base = double.parse(stdin.readLineSync()!);

  stdout.write("Enter the height of the triangle: ");
  double height = double.parse(stdin.readLineSync()!);

  double area = (base * height) / 2;

  print("The area of the triangle with base $base and height $height is $area square units.");
}
